app/[locale]/client/settings/settings-content.tsx

```
@@ -42,49 +42,50 @@
+      title: t('SETTINGS_CARDS.BASIC_INFO.TITLE'),
+      description: t('SETTINGS_CARDS.BASIC_INFO.DESCRIPTION'),
       icon: <FiUser className="w-5 h-5 text-theme-primary-600 dark:text-theme-primary-400" />,
       href: `/client/settings/profile/basic-info`
     },
     {
       id: "security",
-      title: "Security & Password",
-      description: "Change your password, enable two-factor authentication, and manage account security",
+      title: t('SETTINGS_CARDS.SECURITY.TITLE'),
+      description: t('SETTINGS_CARDS.SECURITY.DESCRIPTION'),
       icon: <svg className="w-5 h-5 text-theme-primary-600 dark:text-theme-primary-400" fill="none" stroke="currentColor" viewBox="0 0 24 24"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M9 12l2 2 4-4m5.618-4.016A11.955 11.955 0 0112 2.944a11.955 11.955 0 01-8.618 3.04A12.02 12.02 0 003 9c0 5.591 3.824 10.29 9 11.622 5.176-1.332 9-6.03 9-11.622 0-1.042-.133-2.052-.382-3.016z" /></svg>,
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)